From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@loekd__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@loekd : Thanks for your contribution to the Azure documentation! The author, @chackdan, has been notified to review your proposed change.

ChackDan commented:
#sign-off

PRmerger commented:
Only the author of the article can use the sign off comment to add the ready-to-merge label.

jomolnar commented:
@ChackDan
Your sign off did not work with the comment automation because your GitHub account is listed incorrectly in the article metadata: authors="chackdan"
It needs to match your GitHub account EXACTLY, including capitalization. Please fix your GitHub account in all your articles after this is merged. Thanks.

hpsin committed:
Explain resource parameter

v-alje committed:
Bringing even with master.

hpsin committed:
OpenId Connect fixes

hpsin committed:
Change Client Credentials link

KumudD committed:
fixed table format

KumudD committed:
Merge branch 'release-mvc-load-balancer' of https://github.com/Microsoft/azure-docs-pr into release-mvc-load-balancer

hpsin committed:
Code credentials fixes

itechedit committed:
Merge pull request #37811 from ClareMSYanGit/cyan-release-working-iot-overview

Finalizing Overview services and technologies

hpsin committed:
Acrolinx for Getting Started

BryanLa committed:
rename iot to iot-fundamentals

BryanLa committed:
fix links

v-alje committed:
Bringing even with master.

Commit:
Reverted product names for intro and service and tech pages

hpsin committed:
oauth v2 code

Ja-Dunn committed:
Merge pull request #38318 from ClareMSYanGit/cyan-release-working-iot-overview

Reverted product names for intro and service and tech pages

BryanLa committed:
Merge branch 'release-build-iot-reorg' of https://www.github.com/microsoftdocs/azure-docs-pr into bryanla-release-build-iot-reorg

BryanLa committed:
add iot dir

BryanLa committed:
final updates, fix bc

KumudD committed:
Merge branch 'release-mvc-load-balancer' of https://github.com/Microsoft/azure-docs-pr into release-mvc-load-balancer

BryanLa committed:
final updates, fix bc

KumudD committed:
new tutorial

v-alje committed:
Bringing even with master.

anumjs committed:
Added ADF files

GitHubber17 committed:
Merge pull request #38294 from BryanLa/bryanla-release-build-iot-reorg

Directory reorg

vhorne committed:
mvc-03

vhorne committed:
mvc-04

vhorne committed:
mvc-05

KumudD committed:
fixed merge conflict

vhorne committed:
mvc-06

vhorne committed:
mvc-07

anumjs committed:
tweaked formatting of the document

BryanLa committed:
change service TOC link names to fundamentals

anumjs committed:
Tweaked formatting

anumjs committed:
Removed a duplicated image

v-shils committed:
Merge pull request #38436 from BryanLa/bryanla-release-build-iot-reorg

Change service TOC link names to fundamentals

BryanLa committed:
build out index.yml

BryanLa committed:
Merge branch 'release-build-iot-reorg' of https://www.github.com/microsoftdocs/azure-docs-pr into bryanla-release-build-iot-reorg

KumudD committed:
minor updates

BryanLa committed:
build out index.yml

BryanLa committed:
build out index.yml

hpsin committed:
OBO flow and delete OIDC changes

BryanLa committed:
build out index.yml

BryanLa committed:
build out index.yml

BryanLa committed:
build out index.yml

BryanLa committed:
finish index.yml updates

KumudD committed:
added index art files

BryanLa committed:
finish index.yml updates

BryanLa committed:
remove extraneous CRLF

v-alje committed:
Bringing even with master.

vhorne committed:
mvc-08

vhorne committed:
mvc-09

vhorne committed:
mvc-10

vhorne committed:
mvc-11

BryanLa committed:
fix links

vhorne committed:
Merge branch 'release-mvc-app-gateway' into mvc-11

vhorne committed:
mvc-12

KumudD committed:
Merge branch 'release-mvc-load-balancer' of https://github.com/Microsoft/azure-docs-pr into release-mvc-load-balancer

vhorne committed:
mvc-13

vhorne committed:
mvc-14

vhorne committed:
mvc-15

vhorne committed:
mvc-15

BryanLa committed:
fix links

v-alje committed:
Bringing even with master.

tfosmark committed:
Merge pull request #38412 from vhorne/mvc-01

Mvc 01

tfosmark committed:
Merge pull request #38409 from vhorne/mvc-02

add linked files from previous branch

tfosmark committed:
Merge pull request #38410 from vhorne/mvc-03

mvc-03

tfosmark committed:
Merge pull request #38417 from vhorne/mvc-05

mvc-05

itechedit committed:
Merge pull request #38414 from vhorne/mvc-04

mvc-04

GitHubber17 committed:
Merge pull request #38421 from vhorne/mvc-06

mvc-06

GitHubber17 committed:
Merge pull request #38427 from vhorne/mvc-07

mvc-07

GitHubber17 committed:
Merge pull request #38540 from vhorne/mvc-08

mvc-08

GitHubber17 committed:
Merge pull request #38542 from vhorne/mvc-09

mvc-09

GitHubber17 committed:
Merge pull request #38544 from vhorne/mvc-10

mvc-10

GitHubber17 committed:
Merge pull request #38551 from vhorne/mvc-12

mvc-12

GitHubber17 committed:
Merge pull request #38554 from vhorne/mvc-13

mvc-13

GitHubber17 committed:
Merge pull request #38561 from vhorne/mvc-14

mvc-14

KumudD committed:
addressed blocking issues

KumudD committed:
Merge branch 'release-mvc-load-balancer' of https://github.com/Microsoft/azure-docs-pr into release-mvc-load-balancer

KumudD committed:
updated metadata for tutorials

tfosmark committed:
Merge pull request #37804 from KumudD/release-mvc-load-balancer

Load Balancer - MVC

ktoliver committed:
Merge pull request #38444 from BryanLa/bryanla-release-build-iot-reorg

Bryanla release build iot reorg

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

rboucher committed:
Update azure-diagnostics.md

Can also route to Event hubs.  Added that information.

hpsin committed:
Remove incorrect `query` param info for odic

hpsin committed:
Remove query response_type throughout

hpsin committed:
Implicit Grant updates

hpsin committed:
Minor protocols page update

hpsin committed:
Change JWT website to MSFT owned resource.

hpsin committed:
Update date for tokens and claims

hpsin committed:
Merge branch 'master' into docFixes

hpsin committed:
Redirect oidc changes page

hpsin committed:
Merge branch 'docFixes' of https://github.com/hpsin/azure-docs-pr into docFixes

GitHubber17 committed:
Merge pull request #38563 from vhorne/mvc-15

Mvc 15

vhorne committed:
incorp PR review

hpsin committed:
Fix redirect source_path

hpsin committed:
Fix [!TIP] warning in oidc

hpsin committed:
Add extension for redirect

v-kents committed:
update home page with new services

v-kents committed:
correct integrations etc. links

hpsin committed:
Address feedback

v-kents committed:
change order of items on devtools

tynevi committed:
Merge pull request #38769 from MicrosoftDocs/master

update from master

GitHubber17 committed:
Merge pull request #38685 from vhorne/mvc-16

incorp PR review

KumudD committed:
updated intro para

BryanLa committed:
angelar feedback

KumudD committed:
tweak

Commit:
Merge pull request #38775 from KumudD/release-mvc-load-balancer

MVC update

BryanLa committed:
remove excess brs

BryanLa committed:
remove excess brs

KumudD committed:
added toc and index yml files

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

KumudD committed:
fixed link

KumudD committed:
Merge branch 'release-mvc-load-balancer' of https://github.com/Microsoft/azure-docs-pr into release-mvc-load-balancer

KumudD committed:
tweak

KumudD committed:
edit incorp

anumjs committed:
add ADF private preview information

anumjs committed:
included hierarchy in the toc

anumjs committed:
changing SQL Database to SQL DB to shorten toc title

anumjs committed:
decreased image size

BryanLa committed:
more tweaks

vhorne committed:
Merge branch 'release-mvc-app-gateway' into mvc-11

BryanLa committed:
more tweaks

v-deken committed:
Updated Bing News Search landing page

GitHubber17 committed:
metadata on staging - missing leading hyphens

v-deken committed:
Fixed broken link in intro paragraph

BryanLa committed:
more tweaks - ready for final sign-off

GitHubber17 committed:
Merge pull request #38547 from vhorne/mvc-11

mvc-11

v-deken committed:
Removed row title from Quickstarts, ordered list from Tutorials

v-alje committed:
Bringing even with master.

Saisang committed:
replace msdn.microsoft.com/en-us/ to msdn.microsoft.com/

v-deken committed:
Fixed YAML formatting issue

Saisang committed:
replace technet.microsoft.com/en-us/ to technet.microsoft.com/

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

Saisang committed:
azure.microsoft.com/en-us/ to azure.microsoft.com/ part1

Saisang committed:
azure.microsoft.com/en-us/ to azure.microsoft.com/ part2

v-shils committed:
Merge pull request #38786 from KumudD/release-mvc-load-balancer

MVC TOC and Landing Page for Azure Load Balancer

v-kents committed:
re-add go image

Saisang committed:
docs.microsoft.com/en-us/ to docs.microsoft.com/ part1

Saisang committed:
docs.microsoft.com/en-us/ to docs.microsoft.com/ part2

KumudD committed:
updated metadata

anumjs committed:
change image

anumjs committed:
added a new picture for overview

Commit:
Merge pull request #38901 from KumudD/release-mvc-load-balancer

updated metadata

vhorne committed:
address merge to master issues

KumudD committed:
removed files

tynevi committed:
Merge pull request #38918 from vhorne/mvc-fix

address merge to master issues

v-deken committed:
Fixed YAML issue in Tutorials - awaiting info for fix in Quickstarts

v-deken committed:
Fixed YAML issue in Tutorials - awaiting info for fix in Quickstarts

KumudD committed:
added redirects to deleted files

anumjs committed:
updated overview image

anumjs committed:
Updated image

TrishaMc committed:
Merge pull request #38919 from KumudD/release-mvc-load-balancer

removed files

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

anumjs committed:
tweaked formatting and wording

anumjs committed:
tweaked indentation

anumjs committed:
updated adf image

anumjs committed:
added portal image

anumjs committed:
updated images

daveba committed:
removing private preview articles and updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into remove-msi-pp-docs

anumjs committed:
fixed indentation

daveba committed:
removing private preview articles and updating TOC

v-deken committed:
Testing YAML in Quickstarts

anumjs committed:
fixed indentation

v-deken committed:
Testing YAML in Quickstarts

v-deken committed:
Testing YAML in Quickstarts

rmca14 committed:
Merge pull request #38778 from BryanLa/bryanla-release-build-iot-reorg

AngelaR feedback

anumjs committed:
removed nested bullet points as it is not supported by docs

DCtheGeek committed:
Added link to VM Policy

DCtheGeek committed:
Fix added link

daveba committed:
added missing metadata

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into remove-msi-pp-docs

GitHubber17 committed:
Update active-directory-msi-core-prereqs-ua.md

curtand committed:
[Azure AD could app discovery] update firewall and data attribute

Commit:
fix for git issue 7193

GitHubber17 committed:
Merge pull request #38966 from daveba/remove-msi-pp-docs

[Azure AD MSI ]Remove msi pp docs - update overview topic

MGoedtel committed:
Updated json to address double-quote issue

PRMerger16 committed:
Merge pull request #39010 from curtand/cloudapp0425

[Azure AD cloud app discovery] update firewall and data attribute

Commit:
tocfix01

PRMerger17 committed:
Merge pull request #39012 from davidmu1/gitissue7193

fix for git issue 7193

v-kents committed:
fix typo on ruby sdk

PRMerger18 committed:
Merge pull request #39014 from MGoedtel/UpdateLATemplate425

Updated json to address double-quote issue

PRMerger18 committed:
Merge pull request #38648 from rboucher/patch-6

Update azure-diagnostics.md

miaojiang committed:
minor updates

miaojiang committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into aadwarning

bandersmsft committed:
azure policy - changed ms.author and author metadata

v-alje committed:
Merge pull request #39002 from MicrosoftDocs/release-build-iot-reorg

Release build iot reorg

daveba committed:
removing extra parenthesis from image

tynevi committed:
Merge pull request #38716 from v-kents/1218516

Update home page with new services

tynevi committed:
Merge pull request #38849 from MicrosoftDocs/release-mvc-app-gateway

PR for MVC App Gateway

itechedit committed:
Merge pull request #38853 from v-deken/LandingPage-Search

Updated Bing News Search landing page

billmath committed:
adding public preveiw tag

v-shils committed:
Merge pull request #38888 from Saisang/remove-en-us-info

replace msdn.microsoft.com/en-us/ to msdn.microsoft.com/

Commit:
Added link to gateway transit article.

PRMerger14 committed:
Merge pull request #39025 from daveba/update-msi-overview

[Azure AD MSI] Removing extra parenthesis from image

Commit:
Added link to gateway transit article.

Saisang committed:
Merge branch 'master' into remove-en-us-info-part2

Saisang committed:
Merge branch 'master' into remove-en-us-info-part7

Saisang committed:
Merge branch 'master' into remove-en-us-info-part3

Saisang committed:
Merge branch 'master' into remove-en-us-info-part6

PRMerger19 committed:
Merge pull request #39035 from jimdial/patch-7

Added link to gateway transit article.

v-shils committed:
Merge pull request #38892 from Saisang/remove-en-us-info-part4

azure.microsoft.com/en-us/ to azure.microsoft.com/ part2

Commit:
Edit

v-shils committed:
Merge pull request #38890 from Saisang/remove-en-us-info-part2

replace technet.microsoft.com/en-us/ to technet.microsoft.com/

Commit:
Merge pull request #39030 from billmath/aadchealth

adding public preveiw tag

Saisang committed:
Merge branch 'master' into remove-en-us-info-part3

v-shils committed:
Merge pull request #38895 from Saisang/remove-en-us-info-part7

docs.microsoft.com/en-us/ to docs.microsoft.com/ part1

PRMerger10 committed:
Merge pull request #39033 from jimdial/patch-6

Added link to gateway transit article.

v-shils committed:
Merge pull request #38893 from Saisang/remove-en-us-info-part6

docs.microsoft.com/en-us/ to docs.microsoft.com/ part2

Saisang committed:
Merge branch 'master' into remove-en-us-info-part3

cherylmc committed:
Update vpn-gateway-about-vpngateways.md

tynevi committed:
Revert "Update home page with new services"

tynevi committed:
Merge pull request #39045 from MicrosoftDocs/revert-38716-1218516

Revert "Update home page with new services"

mimckitt committed:
Fixes MicrosoftDocs/azure-docs#7677

If people have configured their default location in the Azure CLI, the az acr create command will honor that, and not the resource group location. I suggest adding --location eastus to the create command just in case

MGoedtel committed:
updated template link references and next steps H2

v-shils committed:
Merge pull request #38891 from Saisang/remove-en-us-info-part3

azure.microsoft.com/en-us/ to azure.microsoft.com/ part1

hpsin committed:
Merge branch 'master' into docFixes

PRMerger20 committed:
Merge pull request #39048 from daveirwin1/master

Deprecating three solutions.

MGoedtel committed:
corrected link paths

miaojiang committed:
fix format

hpsin committed:
Handle #7378

anumjs committed:
Included changes suggested by the reviewer

- Added 'Azure' in the meta data title
- removed bullet point from ADF section. Doesn't have to be a list
- Changed the title of the document

daveba committed:
add breadcrumb for MSI for articles outside of TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into fix-breadcrumb-rendering

Commit:
Merge pull request #39049 from MGoedtel/UpdateLATemplateB425

updated template link references and next steps H2

GitHubber17 committed:
Merge pull request #39022 from bandersmsft/master

azure policy - changed ms.author and author metadata

rmca14 committed:
Merge pull request #39031 from MarkusVi/tocfix01

tocfix01

PRMerger14 committed:
Merge pull request #39044 from cherylmc/patch-1

Update vpn-gateway-about-vpngateways.md

v-alje committed:
Resolving merge conflict

TrishaMc committed:
Merge pull request #38381 from anumjs/SaaS-in-the-box

Added ADF files

v-alje committed:
Merge pull request #38917 from MicrosoftDocs/release-mvc-load-balancer

Release mvc load balancer

tfosmark committed:
Merge pull request #39020 from miaojiang/aadwarning

Aadwarning

TrishaMc committed:
Merge pull request #38280 from hpsin/docFixes

Doc fixes for Build

tfosmark committed:
Merge pull request #39051 from daveba/fix-breadcrumb-rendering

[Azure AD MSI]Add breadcrumb rendering for articles outside MSI content set

Commit:
fix for git issue 7540

PRMerger10 committed:
Merge pull request #39046 from MicahMcKittrick-MSFT/patch-23

Fixes MicrosoftDocs/azure-docs#7677

PRMerger12 committed:
Merge pull request #39063 from davidmu1/gitissue7540

fix for git issue 7540

PRMerger13 committed:
Merge pull request #39003 from DCtheGeek/dmc-policyfixes

Added link to VM Policy from Azure Policy sample (Issue #7534)

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Commit:
fix for git issue 7657

v-alje committed:
Merge pull request #39073 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #39075 from davidmu1/gitissue7657

fix for git issue 7657

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
